lib/questions.ts

```
@@ -0,0 +1,83 @@
+import path from "path";
+import matter from "gray-matter";
+import { remark } from "remark";
+import html from "remark-html";
+
+export interface QuestionData {
+  year: string;
+  day: string;
+  contentHtml: string;
+  title?: string;
+  difficulty?: string;
+  [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)